src/riskMetrics/DynamicRisk.m

```diff
@@ -0,0 +1,72 @@
+function DynamicRisk = DynamicRisk(data)
+%DynamicRisk function that compute the dynamic risk based on current
+%glucose concentration and its rate-of-change, using hyperbolic tangent with
+%pre-defined parameters as amplification function
+%
+%Reference:
+%  - S. Guerra et al., "A Dynamic Risk Measure from Continuous Glucose
+%  Monitoring Data", Diabetes Technol. Ther., 2011. 
+%  doi: https://doi.org/10.1089/dia.2011.0006
+%
+% The function ignores nan values
+% 
+%Input:
+%  - data: a timetable with column 'Time' and 'glucose' containing the
+%  glucose data to analyze (in mg/dL)
+%Output:
+%  - DynamicRisk: dynamic risk value.
+%
+%Preconfitions:
+%  - data must be a timetable having an homogeneous time grid.
+%
+% ------------------------------------------------------------------------
+%
+% This file is part of AGATA.
+%
+% ------------------------------------------------------------------------
+
+
+%Check preconditions
+if(~istimetable(data))
+    error('DynamicRisk: data must be a timetable.');
+end
+if(length(unique(diff(data.Time))) ~= 1)
+    error('DynamicRisk: data must have a homogeneous time grid.')
+end
+
+
+%Compute rate-of-change
+ROC = [0;diff(data.glucose)/minutes(unique(diff(data.Time)))];
+
+%Initialization
+alpha = 1.084;      
+beta = 5.381;       
+gamma = 1.509;
+
+MaximumAmplification = 2.5;
+MaximumDamping = 0.65;
+AmplificationRapidity = 3;
+DRdelta=(MaximumAmplification-MaximumDamping)/2;
+DRbeta=DRdelta+MaximumDamping;
+DRgamma=atanh((1-DRbeta)/DRdelta);
```

Comment:
Please use lower case starting letter + camel case for function and variable names e.g. functionNameHere.